src/components/AuthForm.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {
+  Box,
+  TextField,
+  Button,
+  Typography,
+  FormControlLabel,
+  Checkbox,
+} from "@mui/material";
+
+const AuthForm: React.FC = () => {
+  const [isNew, setIsNew] = React.useState(false);
+
+  const handleChange = (event: {
+    target: { checked: boolean | ((prevState: boolean) => boolean) };
```

Comment:
Please, check for existing types: you can use `React.ChangeEvent<HTMLInputElement>` here.